autoscoper/CMakeLists.txt

```diff
@@ -148,11 +148,12 @@ target_include_directories(autoscoper PUBLIC
   ${Qt5Network_INCLUDE_DIRS}
   ${Qt5Gui_INCLUDE_DIRS})
 
-include(UseGLEW)
-UseGLEW(autoscoper PUBLIC)
-
-include(UseLIBTIFF)
-UseLIBTIFF(autoscoper PUBLIC)
+find_package(GLEW)
+target_link_libraries(autoscoper PUBLIC GLEW::GLEW)
+target_compile_definitions(autoscoper PUBLIC -DUSE_GLEW)
+find_package(TIFF)
+target_link_libraries(autoscoper PUBLIC TIFF::TIFF)
+target_compile_definitions(autoscoper PUBLIC -DUSE_LIBTIFF)
```

Comment:
Since TIFF seems to be used directly only in `libautoscoper` and not `autoscoper`, this should be moved at the top of [libautoscoper/CMakeLists.txt](https://github.com/BrownBiomechanics/Autoscoper/blob/main/libautoscoper/CMakeLists.txt)